docs/bugpattern/XorPower.md

```diff
@@ -0,0 +1,4 @@
+The `^` binary XOR operator is sometimes mistaken for a power operator, but e.g.
+`2 ^ 10` evaluates to `0`, not `4`.
```

Comment:
@cushon I'm not clear on how 2 to the power of 10 evaluates to 4. Did you mean to say that it evaluates to 1024, or am I missing something here? :)

Reply:
@cushon Oops, sorry, I saw you fixed it in the next commit!